fusionauth/resource_fusionauth_tenant_test.go

```diff
@@ -180,7 +180,7 @@ func testTenantAccTestCheckFuncs(
 		// resource.TestCheckResourceAttr(tfResourcePath, "family_configuration.0.#family_request_email_template_id", "UUID"),
 		resource.TestCheckResourceAttr(tfResourcePath, "family_configuration.0.maximum_child_age", "14"),
 		resource.TestCheckResourceAttr(tfResourcePath, "family_configuration.0.minimum_owner_age", "18"),
-		resource.TestCheckResourceAttr(tfResourcePath, "family_configuration.0.parent_email_required", "true"),
+		resource.TestCheckResourceAttr(tfResourcePath, "family_configuration.0.parent_email_required", "false"),
```

Comment:
Think their API has changed now and will return false if a `parent_registration_email_template_id` is not set (hence the change here)